project-ui/package.json

```diff
@@ -11,40 +11,41 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^7.2.3",
-    "@angular/cdk": "^7.3.0",
-    "@angular/common": "~7.2.0",
-    "@angular/compiler": "~7.2.0",
-    "@angular/core": "~7.2.0",
-    "@angular/forms": "~7.2.0",
-    "@angular/material": "^7.3.0",
-    "@angular/platform-browser": "~7.2.0",
-    "@angular/platform-browser-dynamic": "~7.2.0",
-    "@angular/router": "~7.2.0",
-    "@types/chart.js": "^2.7.42",
-    "chart.js": "^2.7.3",
-    "core-js": "^2.5.4",
+    "@angular/animations": "^7.2.4",
+    "@angular/cdk": "^7.3.4",
+    "@angular/common": "^7.2.4",
+    "@angular/compiler": "^7.2.4",
+    "@angular/core": "^7.2.4",
+    "@angular/forms": "^7.2.4",
+    "@angular/material": "^7.3.4",
+    "@angular/platform-browser": "^7.2.4",
+    "@angular/platform-browser-dynamic": "^7.2.4",
+    "@angular/router": "^7.2.4",
+    "@types/chart.js": "^2.7.48",
+    "chart.js": "^2.8.0",
+    "chartjs-plugin-annotation": "^0.5.7",
+    "core-js": "^2.6.4",
     "install": "^0.12.2",
-    "ng2-charts": "^1.6.0",
-    "npm": "^6.7.0",
+    "ng2-charts": "^2.0.3",
+    "npm": "^6.9.0",
     "rxjs": "~6.3.3",
     "tslib": "^1.9.0",
     "zone.js": "~0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.13.0",
-    "@angular/cli": "~7.3.0",
-    "@angular/compiler-cli": "~7.2.0",
-    "@angular/language-service": "~7.2.0",
-    "@types/node": "~8.9.4",
+    "@angular-devkit/build-angular": "^0.13.6",
+    "@angular/cli": "^7.3.6",
+    "@angular/compiler-cli": "^7.2.9",
+    "@angular/language-service": "^7.2.9",
     "@types/jasmine": "~2.8.8",
     "@types/jasminewd2": "~2.0.3",
+    "@types/node": "~8.9.4",
     "codelyzer": "~4.5.0",
     "jasmine-core": "~2.99.1",
     "jasmine-spec-reporter": "~4.2.1",
     "karma": "~3.1.1",
     "karma-chrome-launcher": "~2.2.0",
-    "karma-coverage-istanbul-reporter": "~2.0.1",
+    "karma-coverage-istanbul-reporter": "^2.0.5",
```

Comment:
da fuck is that "karma-coverage-istanbul-reporter"  haha? (rien besoin de changer ici, c'est juste drole)

Reply:
Istanbul c'est un outil de monitoring de code coverage pour nodejs
et Karma c'est un outil de test :P
Je ne me sert  de ni l'un ni l'autre, mais ils sont dans les créations de base des projets angular :p